recipes/pepquery/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "pepquery" %}
+{% set version = "1.1" %}
+{% set sha256 = "c819b0405392f13c7a7bfd6df11c3a639a3fe68c5649a136167aa38f40107358" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/jraysajulga/PepQuery/archive/v{{ version }}.zip
```

Comment:
@jraysajulga is that a fork?
There is also a new version available at http://pepquery.org/data/PepQuery_v1.2.0.tar.gz

Reply:
@bgruening Ah. Yes. I was meaning to update...